lavamoat/build-system/policy-override.json

```diff
@@ -99,7 +99,8 @@
         "@typescript-eslint/parser>@typescript-eslint/scope-manager": true,
         "eslint>eslint-scope": true,
         "eslint>eslint-utils": true,
-        "webpack>eslint-scope": true
+        "webpack>eslint-scope": true,
+        "@babel/eslint-parser>eslint-scope": true
```

Comment:
The same package shouldn't be referenced twice here. It looks like the identifier for this package changed, but the old obsolete identifier was left in the override file.

Reply:
Is there a way to fix that without manually verifying each thing?

Reply:
okay resolving those and rerunning lavamoat:auto and builds to check that it works as expected.

Reply:
I'm guessing this change needs to be reverted now, since the webpack downgrade was reversed.